plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py

```diff
@@ -46,51 +47,60 @@ def get_excluded_ports(self):
         """
         items = self.isolation_mgr.exclude_list.items()
         formatted_items = [f"{item.port_name}: {'infinite' if item.ttl_seconds == 0 else int(max(0, item.remove_time - time.time()))}" for item in items]
-        return '\n'.join(formatted_items) + ('' if not formatted_items else '\n')
+        response = '\n'.join(formatted_items) + ('' if not formatted_items else '\n')
+        return response, HTTPStatus.OK
 
 
     def exclude_ports(self):
         """
         Parse input ports and add them to exclude list (or just update TTL)
-        Input string example: 0c42a10300756a04_1,98039b03006c73ba_2:300
+        Input string example: [["0c42a10300756a04_1"],["98039b03006c73ba_2",300]]
         TTL that follows port name after the colon is optional
         """
-        data_dict = request.form.to_dict()
-        if not data_dict:
-            return "No ports added to exclude list"
-      
+
+        try:
+            pairs = self.get_request_data()
+        except JSONDecodeError:
+            return "Incorrect input format\n", HTTPStatus.BAD_REQUEST
```

Comment:
I think the error string message should be in a constant since it appears more then once.

Reply:
Done